packages/grafana-ui/src/components/ToolbarButton/ToolbarButton.story.tsx

```diff
@@ -89,18 +91,71 @@ BasicWithIcon.args = {
   iconOnly: true,
 };
 
-export const List: ComponentStory<typeof ToolbarButton> = (args) => {
+export const Examples: ComponentStory<typeof ToolbarButton> = (args) => {
```

Comment:
**Nit:** maybe I would add a short comment on why we actually need this

My reasoning would be that it can help future devs better understand why it can be important for us to see multiple examples under a "single" story, and like that it hopefully won't be removed in the future. 

```suggestion
// Used for testing the component during development more easily
export const Examples: ComponentStory<typeof ToolbarButton> = (args) => {
```